bears/jinja2/Jinja2Bear.py

```diff
@@ -0,0 +1,335 @@
+import re
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result, RESULT_SEVERITY
+from coalib.results.Diff import Diff
+
+
+def generate_spacing_diff(file, filename, line, line_number,
+                          match_object, required_spacing):
+    '''
+    Generate a diff for incorrectly spaced control or variable tags.
+
+    :param match_object:     A Match object containing the groups ``open``,
+                             ``close`` containing the opening and closing
+                             delimiters of a Jinja2 tag and ``content``
+                             containing everything in between.
+    :param required_spacing: the number of spaces expected after the open
+                             delimiter and before the close delimiter
```

Comment:
should enclose in double backticks "open" and "close" as you did with the other param